kustomize/commands/edit/remove/removeconfigmap.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 The Kubernetes Authors.
+// Copyright 2022 The Kubernetes Authors.
```

Comment:
nit: copyright on existing files should keep their original year

Reply:
So this file and the next one are net-new to the project. I'm a little confused how this is showing as a change -- perhaps a change between commits?

Reply:
Oh interesting, must be. Could you change it to 2023 in that case? For this and the comment below

Reply:
Sure thing!

Reply:
Updated!